renderer/main.js

```diff
@@ -1124,6 +1129,20 @@ function openTorrentContextMenu (infoHash) {
   var torrentSummary = getTorrentSummary(infoHash)
   var menu = new electron.remote.Menu()
 
+  menu.append(new electron.remote.MenuItem({
```

Comment:
Nitpick, I would probably put these two menu items at the bottom and keep Show In Finder on top